yajsapi/package/index.ts

```diff
@@ -13,6 +13,7 @@ export type RepoOpts = {
   image_hash: string;
   min_mem_gib: number;
   min_storage_gib: number;
+  min_available_logical_cpu_cores?: number;
```

Comment:
I'd change the name to `min_cpu_threads`, as discussed in https://github.com/golemfactory/yapapi/pull/508

Reply:
Done (I reverted the "Rename variable" commit).